views/components/nav.njk

```diff
@@ -13,10 +13,8 @@
 {% macro navSearchForm(isOffset = false) %}
     {% set id = 'global-search-input-' | appendUuid %}
     <form class="global-search{% if isOffset %} global-search--offset{% endif %} js-global-search-form" role="search" action="/search">
-        <label class="u-visually-hidden" for="{{ id }}">
-            {{ __("global.misc.search") }}
-        </label>
         <input
+            aria-label="{{ __("global.misc.search") }}"
```

Comment:
Axe now flags that explicit `<labels>` must be visible, so for a hidden label it's more appropriate to use `aria-label`. TIL